CHANGELOG.md

```diff
@@ -1,3 +1,85 @@
+CHANGES IN VERSION 1.6.4 (from 1.6.3)
+===================================
+
+Appium 1.6.4 fixes numerous issues with the previous releases. This release
+supports iOS 10.3 as well as Android 7.1.
+
+#### General
+* Fixed `UnhandledPromiseRejection` errors when running Appium with Node version
+  7
+* Better indicate missing necessary programs to users
+* Fix session creation logging
+* Fix server shutdown on `SIGINT` and `SIGTERM` signals
+* Ensure that all requests have `application/json` content-type
+
+#### iOS
+* Fix issues with error handling in Safari/Webview handling
+* Increase simulator launch timeout for iOS 10+
+* Better handling of page selection in Safari
+* Fix memory usage issues when device logs get large
+* Add `startIWDP` capability to allow Appium to handle starting/stopping `ios-webkit-debug-proxy`
+  automatically.
+* Fix problem where date returned from device was not parsable
+* Allow custom `SafariLauncher` bundle id to be passed in through `bundleId` capability
+  during real device Safari tests, in case Xcode can no longer build the bundled one
+* Fix logging from real device in the case where the device data/time are different
+  from the server
+
+#### iOS - XCUITest
+* Changed the way the Appium checks that WebDriverAgent is running on the device,
```

Comment:
just a few lines here use past tense, whereas everywhere else uses present tense. maybe change to present to keep it consistent?